### PR TITLE
Fix channel creation transaction flow settings

### PR DESCRIPTION
**Pull request by Cloud11PL**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [x] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

## Description

* fixes an issue where settings for transaction flow in channel create were not saved
* fixes an issue where in channel create initial click on checkbox doesn't do anything
* sorts some props alphabetically for better visibility

### CODE REVIEW

**Review by poulch — APPROVED** [review]

**Review by yellowee — DISMISSED** [review]

File: src/channels/components/ChannelForm/MarkAsPaid.tsx [hunk 1]
```diff
@@ -7,20 +7,20 @@ import { FormattedMessage } from "react-intl";
 import { messages } from "./messages";
 
 interface MarkAsPaidProps {
-  isDiabled: boolean;
```
Comment (poulch) [inline src/channels/components/ChannelForm/MarkAsPaid.tsx:10]: question: why did you remove disabled from props IMO is quite important when loading data?

Comment (Cloud11PL) [reply src/channels/components/ChannelForm/MarkAsPaid.tsx:10]: Because none of the checkboxes in this form have it. Should we disable all checkboxes on loading?

**Review by poulch — APPROVED** [review]

**Review by yellowee — APPROVED** [review]
